.github/workflows/docker.yml

```diff
@@ -14,6 +14,6 @@ jobs:
       - name: Build docker image
         run: docker build . -t pyro-crowdsourcer:py3.8.1-slim
       - name: Run docker container
-        run: docker run -d -p 8001:8501 pyro-crowdsourcer:py3.8.1-slim
+        run: docker run -d -p 8001:80 pyro-crowdsourcer:py3.8.1-slim
```

Comment:
Are you positive? The last line of "app.py" seems to run the app on port 8050 :thinking:

Reply:
This is determined by the command in the `Dockerfile`:

```
CMD ["gunicorn", "-b 0.0.0.0:80", "src.app:server"]
```

With which the app runs on port 80.